pkg/operation/cloudbotanist/alicloudbotanist/controlplane.go

```diff
@@ -129,6 +129,22 @@ func (b *AlicloudBotanist) GenerateKubeSchedulerConfig() (map[string]interface{}
 	return nil, nil
 }
 
+// GenerateETCDStorageClassConfig generates values which are required to create etcd volume storageclass properly.
+func (b *AlicloudBotanist) GenerateETCDStorageClassConfig() map[string]interface{} {
+	return map[string]interface{}{
+		"name":        "etcd-fast",
+		"capacity":    "25Gi",
+		"provisioner": "csi-diskplugin",
+		"parameters": map[string]interface{}{
+			"regionId": b.Shoot.Info.Spec.Cloud.Region,
+			"zoneId":   b.Shoot.Info.Spec.Cloud.Alicloud.Zones[0],
```

Comment:
As discussed via Slack: That's wrong, you are taking the zone of the shoot but the storageclass is in the seed. The AZ is not exposed for a seed. Do you know why the AZ is important for Alicloud  storageclasses here? (cc @jia-jerry)
Does that mean that if I provide zone `a` that no pods with this storageclass could be created in other zones `b`, `c`, etc.?

If we really need it: Can we get to that information by querying the node (I see that every alicloud worker node has the `beta.kubernetes.io/zone=eu-central-1b` label).

Reply:
I checked it further, and zoneID is optional for alicloud. I will push changes with other suggestions addressed.

Reply:
Please remove *region* and *zone*, it is not suggested for such information.